waves/src/WalletInfo.tsx

```diff
@@ -31,12 +31,13 @@ interface WalletInfoProps {
 export default function WalletInfo({ balance }: WalletInfoProps) {
     const { isOpen, onOpen, onClose } = useDisclosure();
     const btnRef = React.useRef(null);
-    const [withDrawAddress, setWithDrawAddress] = React.useState("");
+    const [withdrawAddress, setWithdrawAddress] = React.useState("");
 
-    const handleWithdrawAddress = (event: ChangeEvent<HTMLInputElement>) => setWithDrawAddress(event.target.value);
+    const handleWithdrawAddress = (event: ChangeEvent<HTMLInputElement>) => setWithdrawAddress(event.target.value);
 
-    const withdraw = () => {
-        console.log("Withdrawing to " + withDrawAddress);
+    const withdraw = async () => {
+        let txId = await withdrawAll(withdrawAddress);
+        console.log("Withdrew everything. Resulting txId: {}", txId);
```

Comment:
I guess this can go into a follow-up but I think we should display a little popup and point the user to the transaction on the blockchain explorer. Should probably also be configurable at build-time which explorer it uses.